llvm-external-projects/iree-compiler-api/lib/CAPI/CMakeLists.txt

```diff
@@ -28,6 +28,7 @@ add_mlir_public_c_api_library(IREECompilerAPICompilerCAPI
     iree::compiler::ConstEval
     iree::compiler::InputConversion::MHLO::MHLO
     iree::compiler::InputConversion::TOSA::TOSA
+    iree::compiler::InputConversion::TMTensor::TMTensor
```

Comment:
Is this related to making it optional? This feels like it would always compile. Could you check by inserting a compilation failure in TMTensor and trying to build with cmake without it enabled?

Reply:
Great, thanks for confirming.